openmdao/core/tests/test_impl_comp.py

```diff
@@ -544,93 +546,21 @@ def setUp(self):
         prob.run_model()
 
     def test_list_inputs(self):
-        from openmdao.api import Problem
```

Comment:
These lines that were removed. Are they automatically being prepended back into the doc for the doctest? Every feature code-block is supposed to show the imports and statements leading up to it, so that it can be pasted standalone into an editor. At least that is what I gathered was the standard motivation for these.

Reply:
I wrote these docs and the previous file_wrap docs such that successive examples don't have to repeat all the setup code for every example.. in this case since each example is a single line of code that seemed excessive.. I invite the docs czars to have a look at the docs as built and weigh in on whether or not this is more clear and preferred.

Reply:
Well, I guess this one is up to @JustinSGray .

Reply:
Im ok with the way steve did it. The intention is clear and a user should have no trouble understanding what they are supposed to do. 

Shorter docs are better, given equal clarity. I think this is just as clear